src/components/Postiterie.vue

```diff
@@ -3,22 +3,26 @@
   <div class="container">
 
     <Camera v-on:closeCamera="camera = false" v-on:screenshotTaken="getImageFromScreenshot" v-if="camera" />
-    <TakePhoto v-on:clicked="camera = true" @fileupload="getImage" />
+    
+    <TakePhoto class="mobile" v-on:clicked="camera = true" @fileupload="getImage" />
     <ImportPhoto  @fileupload="getImage" />
+
     <UploadPhoto  v-on:updateGmail="updateGmail" @uploadClick="tracking" @inputChange="updateEmail" v-if="!progress && image && !link"/>
-    <progressBar text-position="middle" text="chargement en cours ..." v-if="progress" size="big" v-bind:val="progressPercentage"></progressBar>
+    <progressBar text-position="middle" text="chargement en cours ..." v-if="progress"  size="big" v-bind:val="progressPercentage"></progressBar>
```

Comment:
Why are some components CamelUpperCased and other camelCased?

Reply:
This one was from a plugin so I kept the original casing. I just realised that Vue did not really care if we changed it so I patched that